libraries/base_service.rb

```diff
@@ -93,37 +94,19 @@ def create_user
         end
       end
 
-      # rubocop:disable Metrics/AbcSize
       def install_proxysql_repository
         url = node['proxysql']['repository']['url']
         name = node['proxysql']['repository']['name']
 
-        case node['platform']
-        when 'rhel', 'centos'
-          execute "rpm -Uhv #{url}" do
-            creates "/etc/yum.repos.d/#{name}"
-          end
-        when 'ubuntu', 'debian'
-          basename = ::File.basename(url)
-          repo_deb = Chef::Config[:file_cache_path] + "/#{basename}"
-
-          # Using execute because apt_update is available only in Chef Client 12.7.
-          execute 'apt_update_for_repo' do
-            command %(apt-get update)
-            action :nothing
-          end
+        package 'findutils' if node['platform_version'].to_i >= 8
 
-          dpkg_package basename do
-            source repo_deb
-            action :nothing
-            notifies :run, 'execute[apt_update_for_repo]', :immediate
-          end
+        execute "rpm -Uhv #{url}" do
+          creates "/etc/yum.repos.d/#{name}"
+          notifies :flush_cache, 'package[proxysql]', :immediately
+        end
 
-          remote_file repo_deb do
-            source url
-            notifies :install, "dpkg_package[#{basename}]", :immediate
-            not_if { ::File.exist?(repo_deb) }
-          end
+        package 'proxysql' do
+          action :nothing
         end
```

Comment:
not intuitive from resource name - do you need package install here or this is used to flush package cache?

Reply:
Only to flush package cache. Will check - maybe this flush can be relocated to where actual install takes place.

Reply:
Did exactly that
https://github.com/vinted/chef-proxysql/pull/55/files#diff-10a18e913f01b118de81cd2d7248a5189c6d1b5fbb2949e40ab381cadd92bda3R313